libs/sdk-backend-tiger/src/convertors/toBackend/AnalyticalDashboardConverter.ts

```diff
@@ -0,0 +1,17 @@
+// (C) 2020 GoodData Corporation
+import { AnalyticalDashboardObject } from "@gooddata/api-client-tiger";
+import { IDashboardDefinition } from "@gooddata/sdk-backend-spi";
+import { cloneWithSanitizedIds } from "./IdSanitization";
+
+export function convertAnalyticalDashboard(
+    dashboard: IDashboardDefinition,
+): AnalyticalDashboardObject.IAnalyticalDashboard {
+    return {
+        analyticalDashboard: {
+            isLocked: dashboard.isLocked,
+            dateFilterConfig: cloneWithSanitizedIds(dashboard.filterContext),
+            filterContext: cloneWithSanitizedIds(dashboard.filterContext), // TODO jak správně zpracovat a typovat?
```

Comment:
The filterContext will be a separate entity in tiger (it is being done at the moment), so there will only be a ref here.